zipline-kotlin-plugin/src/main/kotlin/app/cash/zipline/kotlin/ir.kt

```diff
@@ -366,13 +365,11 @@ fun getOrCreateCompanion(
   enclosing: IrClass,
   irPluginContext: IrPluginContext,
 ): IrClass {
-  val existing = enclosing.declarations.firstOrNull {
-    it is IrClass && it.name.identifier == "Companion"
-  }
-  if (existing != null) return existing as IrClass
+  val existing = enclosing.companionObject()
+  if (existing != null) return existing
 
   val irFactory = irPluginContext.irFactory
-  val anyType = irPluginContext.referenceClass(irPluginContext.irBuiltIns.anyType.classFqName!!)!!
+  val anyType = irPluginContext.referenceClass(irPluginContext.irBuiltIns.anyType.getClass()!!.classId!!)!!
```

Comment:
```suggestion
  val anyType = irPluginContext.referenceClass(StandardClassIds.Any)!!
```
Probably needs an import, too